index.html

```diff
@@ -84,7 +84,254 @@ <h3 class="Upcoming-games-title">MARVEL SNAP</h3>
       </div>
     </div>
   </section>
-  <section class="subscribe-container">
+
+  <section class="Section-update">
+    <div class="Update-mask">
+      <div class="Update-container">
+        <ul class="Update-list">
+          <li>
+            <button>STREAMING ON</button>
+          </li>
+          <li> 
+            <button>MOVIES</button>
+          </li>
+          <li> 
+            <button>COMICS</button>
+          </li>
+        </ul>
+
+        <div class="Update-list-container">
+          <div class="Update-image-container">
+            <img class="Update-image" src="/public/disneyplus.webp" alt="marvel and disney plus">
+          </div>
+          <div class="Update-text">
+            <h4>ABOUT US</h4>
+          <p>MARVEL CORPORATE INFORMATION</p>
+          <p>
+            Marvel Entertainment, LLC, a wholly-owned subsidiary of The Walt Disney Company, is one of the world's most prominent character-based entertainment companies, built on a proven library of more than 8,000 characters featured in a variety of media over seventy-five years. Marvel utilizes its character franchises in entertainment, licensing and publishing.
+          </p>
+          <input class="button" type="button" value="Buy Now" onclick="msg()" />
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="Section-gallery">
+    <div class="Gallery-container">
+      <h2>Marvel Gallery</h2>
+      <div class="Gallery-image-split">
+        <div class="Gallery-image-left">
+          <div class="Gallery-left-pics">
+            <img src="/public/helicarrier.jpeg" alt="">
```

Comment:
Every **img** tag should have the alt description

Reply:
👍🏼